src/Splunk.OpenTelemetry.AutoInstrumentation.Plugin/Configuration/CompositeConfigurationSource.cs

```diff
@@ -0,0 +1,117 @@
+﻿// <copyright file="CompositeConfigurationSource.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Splunk.OpenTelemetry.AutoInstrumentation.Plugin.Configuration;
```

Comment:
Looks like a copy from upstream repo. If so, we need add some headers here.

It is applicable for most of the newly added files.

@dszmigielski could you please advice?

Reply:
If its a copy from otel then we need to add both our header as well as otel one.
Example:
https://github.com/signalfx/splunk-otel-dotnet/blob/main/test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/Helpers/EnvironmentHelper.cs